### PR TITLE
Quick Tags UI update

### PR DESCRIPTION
**Pull request by shutterspeak**
Update of the Quick Tags UI to better comply with Tumblr's design guidelines. Photo:
image

### CODE REVIEW

File: Extensions/replyviewer.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ XKit.extensions.replyviewer = new Object({
...
+	button_icon:" data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAAAXNSR0IArs4c6QAAAnhJREFUOBGlVP9rUlEUv++hmWsrp0iJ+0GyIBFEiUmQNAghif0wnPir+ZOQqEh/iJP93A8F0kiELAqhsRIMIRWFkIFgY4mYIKHNRvh1n/PgyTakzA6cd+6955zPveecD4+xOWU8HnPTUvlph7OcRaPRxtbW1pPzwFNvmQUwEomMKY7juA9SqdTj9/trwp4+uOktbnpI63kEoG2e5x8Eg8HPQsn/A0YPQP4C9BKtJfQRJRwOz9wCsWTkNqCboVAoSzhnAOngXwSlZuVy+abP5yNQQeYGRM8eaTSaF263uyeCkZ0bECU+E4G2t7fv0zoQCOzNDSiCkR0Oh0kMhWh0WST2ETakNDEH9CW0Dv0NPYA+hVrJP03gW8T5EvmEF0okko2VlZVlOFKj0ehmLper7+/vc91ulykUCs5sNl83Go2v4d9FzmMM4yclTxOBJgi8Bef7Wq1WTSaTd5Ag1Wq1vFKpZM1mc9xoNMYymezI6XR+UavV1xBrR8yhCChSiGjHA+wGHLuZTOYwkUis6XS6C16vlzcYDIxeaDKZOI/Hw6tUqqVYLGYrFot1xH9Enk4EPG2ph68KhcLXfD5/12azsfX1da7X67FUKsUqlYpgkcxcLhe/urrK0un0WqlU+oa8NzhfOA1Ga6GH/X7/ynnHn/btdvteq9ViOzs7v1DumVAC3LBarZ86nc4AZUvQL2a325nD4WDVapXp9Xr6o7B4PM7q9Tq1gFksFob2sMFgMAFDzDvaTIaC6e6Vy+WLKGmZHJg6m3UoFC+KAEgb9OMqzPPj4+Pb2Wx2ERP/gaEoQZvvoM0BaGPAK/5KGxF4YgFMxI5DZyb2JBmLE4DNNuBNd2zuAAAAAElFTkSuQmCC",
```
Comment (nightpool) [inline Extensions/replyviewer.js]: ??

Comment (shutterspeak) [inline Extensions/replyviewer.js]: The icon is based on a conversation we had a while ago, if that's the issue. Or is it something else? I just went back and fixed the formatting.

File: Extensions/quick_tags.js [hunk 2]
```diff
@@ -69,7 +68,9 @@ XKit.extensions.quick_tags = new Object({
 		this.running = true;
 
 		XKit.tools.init_css("quick_tags");
-		
+
+		if (!$(".post.post_full, .post.post_brick").length) { return; }
```
Comment (hobinjk) [inline Extensions/quick_tags.js]: Duplicate of line 74 below

File: Extensions/quick_tags.css [hunk 3]
```diff
@@ -1,14 +1,13 @@
 #xkit-quick-tags-window {
 
 	width: 210px;
-	badckground: rgba(222,224,230,0.47);
+	badckground: rgb(255,255,255);
```
Comment (hobinjk) [inline Extensions/quick_tags.css]: Typo persists

**Pull request merged**